dtfs-central-api/src/v1/controllers/portal/deal/update-deal.controller.js

```diff
@@ -65,74 +67,78 @@ const updateDealEditedByPortal = async (dealId, user) => {
 };
 exports.updateDealEditedByPortal = updateDealEditedByPortal;
 
-const updateDeal = async (dealId, dealChanges, user, existingDeal, routePath) => {
+const updateDeal = async (dealId, dealChanges, user, existingDeal, routePath, auditDetails) => {
```

Comment:
This is exported & I think used in a couple other places too (if it isn't it shouldn't be exported!)